go_modules/spec/dependabot/go_modules/file_updater/go_mod_updater_spec.rb

```diff
@@ -421,7 +421,7 @@
       let(:previous_requirements) { [] }
 
       it "raises the correct error" do
-        error_class = Dependabot::DependencyFileNotResolvable
+        error_class = Dependabot::DependabotError
```

Comment:
So this makes the tests pass, but I kinda wonder if the problem isn't the test, but rather some code needs to be added to transform the `go get` failure from a catch-all `DependabotError` to a `DependencyFileNotResolvable` error... Any idea where that should happen?

Reply:
NVM, I found the problematic regex that needed updating, so I put back the test to before.